cmd/flux/version.go

```diff
@@ -12,6 +13,10 @@ func getVersion(input string) (string, error) {
 		return rootArgs.defaults.Version, nil
 	}
 
+	if !strings.HasPrefix(input, "v") {
+		input = "v" + input
+	}
+
```

Comment:
So basically this is the only change that I should keep @stefanprodan? (If you are interested in this PR)

Reply:
I'm for bailing out out when the version is not found instead of defaulting to latest.

Reply:
How do you propose we do that? If bootstrap and install must work if external calls to GitHub fail, we'd need to do something like this:

Abort bootstrap/install if:
* Version check against GH succeeded, but version not found.

Do not abort bootstrap/install if:
* Version check against GH succeeded, version found.
* Version check against GH failed.

I also noticed that there are discrepancies between how install and bootstrap work at the moment, when passing in different kinds of versions:

**Install:**

Version check fails, install aborted (I guess this goes against "should work in an air gapped env"? 🤔):
0.xxx.0
v0.xxx.0

Version check succeeds, installs latest:
0.13.1

Version check succeeds, installs correct:
v0.13.1

**Bootstrap:**

Version check fails, bootstraps latest:
0.xxx.0
v0.xxx.0

Version check succeeds, bootstraps latest:
0.13.1

Version check succeeds, boostraps correct:
v0.13.1

So I'm not really sure what to do? 😄

Reply:
We could create a custom flag that validates the string is a valid semver and that it has the v prefix.

Reply:
A valid semver could still be a "not found" version, so that would still default to latest, which from a users perspective is a bit weird I think.

What I would expect as a user:
* Invalid version -> Install/Boostrap fails
* No version set -> Default to latest

But if we make that change, I guess it would be a breaking change for air gapped envs (force them to not set a version, so that no external call is made).